nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/zookeeper.properties

```diff
@@ -27,6 +27,28 @@ tickTime=2000
 dataDir=./state/zookeeper
 autopurge.snapRetainCount=30
 
+# Embedded/distributed ZK TLS connection support can be activated by setting these properties at minimum:
+#
+# secureClientPort=2281
+# serverCnxnFactory=org.apache.zookeeper.server.NettyServerCnxnFactory
+
+# Most TLS configurations will set these values as well:
+#
+# ssl.keyStore.location=/example/path/to/key-store.jks
+# ssl.keyStore.password=change this value to the actual value in your installation
+# ssl.trustStore.location=/example/path/to/trust-store.jks
+# ssl.trustStore.password=change this value to the actual value in your installation
+# ssl.hostnameVerification=false
```

Comment:
Let's remove this, or note that you would only do it for testing.

Reply:
I'll add a note to that effect.